### PR TITLE
Adds GUSD-A

### PR DESCRIPTION
**Pull request by godsflaw**
# Description

Spell Description:

COB Team Name or Author(s): @hexonaut @godsflaw @brianmcmichael @lucas-manuel 

# Contribution Checklist

- [x] First commit title starts with COB Team Name and Collateral type `GUSD-A`
- [x] Code approved
- [x] Tests approved
- [x] CI Tests pass

# Checklist

- [x] Every contract variable/method declared as public/external private/internal
- [x] Verify expiration (`4 days + 2 hours` monthly and `30 days` for the rest)
- [x] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [x] Validate all addresses used are in Kovan changelog
- [x] Deploy spell to kovan `SOLC_FLAGS="--optimize --optimize-runs=1" dapp --use solc:0.5.12 build --extract && dapp create DssSpell --gas=5000000 --gas-price="$(seth --to-wei 3 "gwei")"`
- [x] Ensure contract is verified on `kovan` etherscan
- [x] Change test to use kovan spell address and deploy timestamp
- [x] Keep `Kovan-DssSpell.sol` and `Kovan-DssSpell.t.sol` the same, but make a copy in `archive`
- [x] `squash and merge` this PR

### CODE REVIEW

**Review by lucas-manuel — APPROVED** [review]
LGTM, had one comment.

File: src/Kovan-DssSpell.sol [hunk 1]
```diff
@@ -16,18 +16,50 @@
...
+    address MCD_JUG;
+    address MCD_SPOT;
+    address MCD_POT;
+    address MCD_END;
+    address FLIPPER_MOM;
+    address OSM_MOM;
+    address ILK_REGISTRY;
+
+    address FAUCET;
+    */
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply src/Kovan-DssSpell.sol:55]: yes

[the remark this answers is not in the document]
Comment (brianmcmichael) [reply src/Kovan-DssSpell.sol]: done

**Comment by brianmcmichael** [comment]
https://kovan.etherscan.io/address/0xcd6359944e35bfde617fbee6ac663a2ef13a34a2

**Pull request merged**